### PR TITLE
Export common-styles sass files that are copied into react package

### PR DESCRIPTION
**Pull request by micahgodbolt**
The original goal was to move sass into common-styles, but to keep backwards compatible by copying the files into @fluentui/react published packages. Due to a bug those files were not properly being copied over. Now that [this bug is fixed](https://github.com/microsoft/fluentui/pull/24215), we need to explicitly export them.

### CODE REVIEW

**Comment by fabricteam** [comment]
## 📊 Bundle size report

<sub>🤖 This report was generated against <a href='https://github.com/microsoft/fluentui/commit/a66448342c1f3e6a6911404486aef904529988c4'>a66448342c1f3e6a6911404486aef904529988c4</a></sub>
<div id="bundleSizeComment9423"/>

**Comment by fabricteam** [comment]
## [Perf Analysis (`@fluentui/react`)](https://github.com/microsoft/fluentui/wiki/Perf-Testing)

  <table>
  <tr>
    <th>Scenario</th>
    <th>Render type</th>
    <th>
      <a href="https://github.com/microsoft/fluentui/wiki/Perf-Testing#why-are-results-listed-in-ticks-instead-of-time-units">Master Ticks</a>
    </th>
    <th>
      <a href="https://github.com/microsoft/fluentui/wiki/Perf-Testing#why-are-results-listed-in-ticks-instead-of-time-units">PR Ticks</a>
    </th>
    <th>Iterations</th>
    <th>Status</th>
  </tr><tr>
            <td>buttonNative</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/24361/perf-test/buttonNative-mount_base.html">155</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/24361/perf-test/buttonNative-mount.html">160</a></td>
            <td>5000</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/24361/perf-test/buttonNative-mount.regression.txt">Possible regression</a></td>
           </tr></table><details><summary>All results</summary><p>
  <table>
  <tr>
    <th>Scenario</th>
    <th>Render type</th>
    <th>
      <a href="https://github.com/microsoft/fluentui/wiki/Perf-Testing#why-are-results-listed-in-ticks-instead-of-time-units">Master Ticks</a>
    </th>
    <th>
      <a href="https://github.com/microsoft/fluentui/wiki/Perf-Testing#why-are-results-listed-in-ticks-instead-of-time-units">PR Ticks</a>
    </th>
    <th>Iterations</th>
    <th>Status</th>
  </tr><tr>
            <td>BaseButton</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/24361/perf-test/BaseButton-mount_base.html">1093</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/24361/perf-test/BaseButton-mount.html">1095</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>Breadcrumb</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/24361/perf-test/Breadcrumb-mount_base.html">3062</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/24361/perf-test/Breadcrumb-mount.html">3033</a></td>
            <td>1000</td>
            <td></td>
           </tr>
<tr>
            <td>Checkbox</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/24361/perf-test/Checkbox-mount_base.html">3027</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/24361/perf-test/Checkbox-mount.html">3125</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>CheckboxBase</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/24361/perf-test/CheckboxBase-mount_base.html">2606</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/24361/perf-test/CheckboxBase-mount.html">2622</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>ChoiceGroup</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/24361/perf-test/ChoiceGroup-mount_base.html">5501</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/24361/perf-test/ChoiceGroup-mount.html">5444</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>ComboBox</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/24361/perf-test/ComboBox-mount_base.html">1150</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/24361/perf-test/ComboBox-mount.html">1080</a></td>
            <td>1000</td>
            <td></td>
           </tr>
<tr>
            <td>CommandBar</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/24361/perf-test/CommandBar-mount_base.html">11478</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/24361/perf-test/CommandBar-mount.html">11593</a></td>
            <td>1000</td>
            <td></td>
           </tr>
<tr>
            <td>ContextualMenu</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/24361/perf-test/ContextualMenu-mount_base.html">12851</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/24361/perf-test/ContextualMenu-mount.html">12635</a></td>
            <td>1000</td>
            <td></td>
           </tr>
<tr>
            <td>DefaultButton</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/24361/perf-test/DefaultButton-mount_base.html">1326</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/24361/perf-test/DefaultButton-mount.html">1344</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>DetailsRow</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/24361/perf-test/DetailsRow-mount_base.html">4422</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/24361/perf-test/DetailsRow-mount.html">4329</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>DetailsRowFast</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/24361/perf-test/DetailsRowFast-mount_base.html">4413</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/24361/perf-test/DetailsRowFast-mount.html">4449</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>DetailsRowNoStyles</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/24361/perf-test/DetailsRowNoStyles-mount_base.html">4208</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/24361/perf-test/DetailsRowNoStyles-mount.html">4311</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>Dialog</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/24361/perf-test/Dialog-mount_base.html">3177</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/24361/perf-test/Dialog-mount.html">3196</a></td>
            <td>1000</td>
            <td></td>
           </tr>
<tr>
            <td>DocumentCardTitle</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/24361/perf-test/DocumentCardTitle-mount_base.html">201</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/24361/perf-test/DocumentCardTitle-mount.html">189</a></td>
            <td>1000</td>
            <td></td>
           </tr>
<tr>
            <td>Dropdown</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/24361/perf-test/Dropdown-mount_base.html">3860</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/24361/perf-test/Dropdown-mount.html">3820</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>FocusTrapZone</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/24361/perf-test/FocusTrapZone-mount_base.html">2119</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/24361/perf-test/FocusTrapZone-mount.html">2164</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>FocusZone</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/24361/perf-test/FocusZone-mount_base.html">2126</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/24361/perf-test/FocusZone-mount.html">2077</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>IconButton</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/24361/perf-test/IconButton-mount_base.html">2097</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/24361/perf-test/IconButton-mount.html">2099</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>Label</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/24361/perf-test/Label-mount_base.html">388</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/24361/perf-test/Label-mount.html">390</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>Layer</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/24361/perf-test/Layer-mount_base.html">3671</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/24361/perf-test/Layer-mount.html">3700</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>Link</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/24361/perf-test/Link-mount_base.html">553</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/24361/perf-test/Link-mount.html">564</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>MenuButton</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/24361/perf-test/MenuButton-mount_base.html">1749</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/24361/perf-test/MenuButton-mount.html">1779</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>MessageBar</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/24361/perf-test/MessageBar-mount_base.html">2371</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/24361/perf-test/MessageBar-mount.html">2353</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>Nav</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/24361/perf-test/Nav-mount_base.html">3818</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/24361/perf-test/Nav-mount.html">3762</a></td>
            <td>1000</td>
            <td></td>
           </tr>
<tr>
            <td>OverflowSet</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/24361/perf-test/OverflowSet-mount_base.html">1244</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/24361/perf-test/OverflowSet-mount.html">1236</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>Panel</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/24361/perf-test/Panel-mount_base.html">2465</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/24361/perf-test/Panel-mount.html">2467</a></td>
            <td>1000</td>
            <td></td>
           </tr>
<tr>
            <td>Persona</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/24361/perf-test/Persona-mount_base.html">1124</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/24361/perf-test/Persona-mount.html">1156</a></td>
            <td>1000</td>
            <td></td>
           </tr>
<tr>
            <td>Pivot</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/24361/perf-test/Pivot-mount_base.html">1624</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/24361/perf-test/Pivot-mount.html">1689</a></td>
            <td>1000</td>
            <td></td>
           </tr>
<tr>
            <td>PrimaryButton</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/24361/perf-test/PrimaryButton-mount_base.html">1534</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/24361/perf-test/PrimaryButton-mount.html">1523</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>Rating</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/24361/perf-test/Rating-mount_base.html">9260</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/24361/perf-test/Rating-mount.html">9297</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>SearchBox</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/24361/perf-test/SearchBox-mount_base.html">1569</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/24361/perf-test/SearchBox-mount.html">1563</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>Shimmer</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/24361/perf-test/Shimmer-mount_base.html">2833</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/24361/perf-test/Shimmer-mount.html">2896</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>Slider</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/24361/perf-test/Slider-mount_base.html">2275</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/24361/perf-test/Slider-mount.html">2208</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>SpinButton</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/24361/perf-test/SpinButton-mount_base.html">5787</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/24361/perf-test/SpinButton-mount.html">5730</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>Spinner</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/24361/perf-test/Spinner-mount_base.html">487</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/24361/perf-test/Spinner-mount.html">471</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>SplitButton</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/24361/perf-test/SplitButton-mount_base.html">3668</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/24361/perf-test/SplitButton-mount.html">3775</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>Stack</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/24361/perf-test/Stack-mount_base.html">602</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/24361/perf-test/Stack-mount.html">611</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>StackWithIntrinsicChildren</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/24361/perf-test/StackWithIntrinsicChildren-mount_base.html">2763</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/24361/perf-test/StackWithIntrinsicChildren-mount.html">2753</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>StackWithTextChildren</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/24361/perf-test/StackWithTextChildren-mount_base.html">6357</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/24361/perf-test/StackWithTextChildren-mount.html">6355</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>SwatchColorPicker</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/24361/perf-test/SwatchColorPicker-mount_base.html">13295</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/24361/perf-test/SwatchColorPicker-mount.html">13127</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>TagPicker</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/24361/perf-test/TagPicker-mount_base.html">3116</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/24361/perf-test/TagPicker-mount.html">3037</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>TeachingBubble</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/24361/perf-test/TeachingBubble-mount_base.html">97632</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/24361/perf-test/TeachingBubble-mount.html">99948</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>Text</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/24361/perf-test/Text-mount_base.html">513</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/24361/perf-test/Text-mount.html">498</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>TextField</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/24361/perf-test/TextField-mount_base.html">1602</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/24361/perf-test/TextField-mount.html">1615</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>ThemeProvider</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/24361/perf-test/ThemeProvider-mount_base.html">1373</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/24361/perf-test/ThemeProvider-mount.html">1343</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>ThemeProvider</td>
            <td>virtual-rerender</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/24361/perf-test/ThemeProvider-virtual-rerender_base.html">734</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/24361/perf-test/ThemeProvider-virtual-rerender.html">762</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>ThemeProvider</td>
            <td>virtual-rerender-with-unmount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/24361/perf-test/ThemeProvider-virtual-rerender-with-unmount_base.html">1966</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/24361/perf-test/ThemeProvider-virtual-rerender-with-unmount.html">2025</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>Toggle</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/24361/perf-test/Toggle-mount_base.html">937</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/24361/perf-test/Toggle-mount.html">943</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>buttonNative</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/24361/perf-test/buttonNative-mount_base.html">155</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/24361/perf-test/buttonNative-mount.html">160</a></td>
            <td>5000</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/24361/perf-test/buttonNative-mount.regression.txt">Possible regression</a></td>
           </tr></table></p></details>

<div id="perfComment9423"/>

File: change/@fluentui-react-5d5942dd-111c-4d36-862d-015a1c872463.json [hunk 1]
```diff
@@ -0,0 +1,7 @@
+{
+  "type": "minor",
+  "comment": "explicit exports for common-styles sass files copied into react package",
```
Comment (sopranopillow) [inline change/@fluentui-react-5d5942dd-111c-4d36-862d-015a1c872463.json:3]: nit

```suggestion
  "comment": "feat: Explicit exports for common-styles sass files copied into react package",
```

File: packages/react/package.json [hunk 2]
```diff
@@ -69,6 +69,42 @@
     "./dist/sass/References": {
       "sass": "./dist/sass/_References.scss"
     },
+    "./dist/sass/common": {
+      "sass": "./dist/sass/_common.scss"
+    },
+    "./dist/sass/constants": {
+      "sass": "./dist/sass/_constants.scss"
+    },
```
Comment (sopranopillow) [inline packages/react/package.json:77]: I see the underscore in References, I think the only one that doesn't have it is Theming. IMO I would keep them, but idk if there's a reason why to remove them?

**Review by sopranopillow — APPROVED** [review]

File: packages/react/package.json (hunk 2 above)
Comment (micahgodbolt) [reply packages/react/package.json:77]: after talking with David, he said he's comfortable here simply using a star export to make the entire sass folder accessible. That also means keeping the file names as is, without removing any underscores.

**Review by dzearing — APPROVED** [review]

File: packages/react/package.json [hunk 3]
```diff
@@ -66,9 +66,7 @@
     "react-dom": ">=16.8.0 <18.0.0"
   },
   "exports": {
-    "./dist/sass/References": {
-      "sass": "./dist/sass/_References.scss"
```
Comment (micahgodbolt) [inline packages/react/package.json:70]: this never actually worked. 'sass' is not a valid condition

**Pull request merged**